### ISSUE TITLE
Tracebacks can't load code blocks with html references

### ISSUE THREAD

**Issue by loftusa** [microsoft/vscode-jupyter#14184]
## Environment data

-   VS Code version: 1.82.0-insider (Universal). Commit: a0377f0c51dbb2d3188565cdf35e89929f864e65
-   Jupyter Extension version (available under the Extensions sidebar): v2023.8.1002381100
-   Python Extension version (available under the Extensions sidebar): v2023.15.12361009
-   OS (Windows | Mac | Linux distro) and version: Mac Ventura 13.4
-   Python and/or Anaconda version: micromamba python version 3.10.12
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda/mamba
-   Jupyter server running: Local + Remote (broken on both)

## Expected behaviour
Tracebacks looks reasonable

## Actual behaviour
<img width="524" alt="Screenshot 2023-08-24 at 2 31 11 PM" src="https://github.com/microsoft/vscode-jupyter/assets/12386450/c954092d-6844-4098-8665-295f4937974f">

Copy-pasteable code:
```
ImportError                               Traceback (most recent call last)
[/Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/_.py](https://file+.vscode-resource.vscode-cdn.net/Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/_.py) in line 2
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/_.py?line=0'>1</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/_.py?line=0%27%3E1%3C/a%3E) #%%
----> <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/_.py?line=1'>2</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/_.py?line=1%27%3E2%3C/a%3E) from utils import DataLoader, DataPuller
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/_.py?line=2'>3</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/_.py?line=2%27%3E3%3C/a%3E) import pandas as pd
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/_.py?line=3'>4</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/_.py?line=3%27%3E4%3C/a%3E) from pathlib import Path

File [~/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py:3](https://file+.vscode-resource.vscode-cdn.net/Users/alex/creyon/experiments/2023-00079_calciumexp/~/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py:3)
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=0'>1</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=0%27%3E1%3C/a%3E) from collections import defaultdict
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=1'>2</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=1%27%3E2%3C/a%3E) from tqdm import tqdm
----> <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=2'>3</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=2%27%3E3%3C/a%3E) from caiman.source_extraction.cnmf.deconvolution import (
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=3'>4</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=3%27%3E4%3C/a%3E)     constrained_oasisAR2,
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=4'>5</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=4%27%3E5%3C/a%3E)     estimate_parameters,
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=5'>6</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=5%27%3E6%3C/a%3E) )
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=6'>7</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=6%27%3E7%3C/a%3E) from peakutils import baseline
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=7'>8</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/notebooks/utils.py?line=7%27%3E8%3C/a%3E) import sys

File [~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/__init__.py:3](https://file+.vscode-resource.vscode-cdn.net/Users/alex/creyon/experiments/2023-00079_calciumexp/~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/__init__.py:3)
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/__init__.py?line=0'>1</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/__init__.py?line=0%27%3E1%3C/a%3E) #!/usr/bin/env python
----> <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/__init__.py?line=2'>3</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/__init__.py?line=2%27%3E3%3C/a%3E) from . import utilities
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/__init__.py?line=3'>4</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/__init__.py?line=3%27%3E4%3C/a%3E) from . import cnmf
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/__init__.py?line=4'>5</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/__init__.py?line=4%27%3E5%3C/a%3E) from . import deconvolution

File [~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py:37](https://file+.vscode-resource.vscode-cdn.net/Users/alex/creyon/experiments/2023-00079_calciumexp/~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py:37)
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=32'>33</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=32%27%3E33%3C/a%3E) from typing import List
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=33'>34</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=33%27%3E34%3C/a%3E) # https://github.com/constantinpape/z5/issues/146
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=34'>35</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=34%27%3E35%3C/a%3E) #import z5py
---> <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=36'>37</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=36%27%3E37%3C/a%3E) from .initialization import greedyROI
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=37'>38</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=37%27%3E38%3C/a%3E) from ...base.rois import com
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=38'>39</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py?line=38%27%3E39%3C/a%3E) from ...mmapping import parallel_dot_product, load_memmap

File [~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py:37](https://file+.vscode-resource.vscode-cdn.net/Users/alex/creyon/experiments/2023-00079_calciumexp/~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py:37)
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py?line=33'>34</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py?line=33%27%3E34%3C/a%3E) from typing import List
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py?line=34'>35</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py?line=34%27%3E35%3C/a%3E) import warnings
---> <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py?line=36'>37</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py?line=36%27%3E37%3C/a%3E) import caiman
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py?line=37'>38</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py?line=37%27%3E38%3C/a%3E) from .deconvolution import constrained_foopsi
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py?line=38'>39</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/initialization.py?line=38%27%3E39%3C/a%3E) #from .utilities import fast_graph_Laplacian_patches

File [~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py:4](https://file+.vscode-resource.vscode-cdn.net/Users/alex/creyon/experiments/2023-00079_calciumexp/~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py:4)
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py?line=0'>1</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py?line=0%27%3E1%3C/a%3E) #!/usr/bin/env python
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py?line=2'>3</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py?line=2%27%3E3%3C/a%3E) import pkg_resources
----> <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py?line=3'>4</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py?line=3%27%3E4%3C/a%3E) from .base.movies import movie, load, load_movie_chain, _load_behavior
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py?line=4'>5</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py?line=4%27%3E5%3C/a%3E) from .base.timeseries import concatenate
      <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py?line=5'>6</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/__init__.py?line=5%27%3E6%3C/a%3E) from .cluster import start_server, stop_server

File [~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py:57](https://file+.vscode-resource.vscode-cdn.net/Users/alex/creyon/experiments/2023-00079_calciumexp/~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py:57)
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py?line=53'>54</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py?line=53%27%3E54%3C/a%3E) from .traces import trace
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py?line=55'>56</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py?line=55%27%3E56%3C/a%3E) from ..mmapping import load_memmap
---> <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py?line=56'>57</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py?line=56%27%3E57%3C/a%3E) from ..utils import visualization
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py?line=57'>58</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py?line=57%27%3E58%3C/a%3E) from .. import summary_images as si
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py?line=58'>59</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/base/movies.py?line=58%27%3E59%3C/a%3E) from ..motion_correction import apply_shift_online, motion_correct_online

File [~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py:33](https://file+.vscode-resource.vscode-cdn.net/Users/alex/creyon/experiments/2023-00079_calciumexp/~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py:33)
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py?line=29'>30</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py?line=29%27%3E30%3C/a%3E) import functools as fct
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py?line=31'>32</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py?line=31%27%3E32%3C/a%3E) from ..base.rois import com
---> <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py?line=32'>33</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py?line=32%27%3E33%3C/a%3E) from ..summary_images import local_correlations
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py?line=34'>35</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py?line=34%27%3E35%3C/a%3E) try:
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py?line=35'>36</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/utils/visualization.py?line=35%27%3E36%3C/a%3E)     cv2.setNumThreads(0)

File [~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py:22](https://file+.vscode-resource.vscode-cdn.net/Users/alex/creyon/experiments/2023-00079_calciumexp/~/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py:22)
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=19'>20</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=19%27%3E20%3C/a%3E) import caiman as cm
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=20'>21</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=20%27%3E21%3C/a%3E) from caiman.source_extraction.cnmf.pre_processing import get_noise_fft
---> <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=21'>22</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=21%27%3E22%3C/a%3E) from caiman.source_extraction.cnmf.utilities import get_file_size
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=23'>24</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=23%27%3E24%3C/a%3E) def max_correlation_image(Y, bin_size: int = 1000, eight_neighbours: bool = True, swap_dim: bool = True) -> np.ndarray:
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=24'>25</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=24%27%3E25%3C/a%3E)     """Computes the max-correlation image for the input dataset Y with bin_size
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=25'>26</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=25%27%3E26%3C/a%3E) 
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=26'>27</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=26%27%3E27%3C/a%3E)     Args:
   (...)
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=43'>44</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=43%27%3E44%3C/a%3E)             max correlation image
     <a href='[file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=44'>45</a>](file:///Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/summary_images.py?line=44%27%3E45%3C/a%3E)     """

ImportError: cannot import name 'get_file_size' from partially initialized module 'caiman.source_extraction.cnmf.utilities' (most likely due to a circular import) (/Users/alex/creyon/experiments/2023-00079_calciumexp/pipelines/CaImAn/caiman/source_extraction/cnmf/utilities.py)
```

## Steps to reproduce:
Not sure how to reproduce from scratch - this started happening for me about a week ago. Tracebacks are broken across all files, with both `.py` files and `.ipynb` files (`.py` using jupyter interactive mode with the `#%%` functionality)

## Logs

```
14:26:09.655 [warn] StdErr from Kernel Process ~/mambaforge/lib/python3.10/site-packages/traitlets/traitlets.py:2499: FutureWarning: Supporting extra quotes around Bytes is deprecated in traitlets 5.0. Use '9ad056a4-187c-4f90-b1af-29de322e4648' instead of 'b"9ad056a4-187c-4f90-b1af-29de322e4648"'.
  warn(
14:26:09.940 [info] Generated code for 1 = <ipython-input-1-944b43cc588e> with 4 lines
14:26:09.957 [info] Kernel acknowledged execution of cell 1 @ 1692912369956
14:26:50.471 [info] End cell 1 execution @ 1692912410467, started @ 1692912369956, elapsed time = 40.511s
14:26:50.471 [warn] Cancel all remaining cells due to cancellation or failure in execution
14:27:15.058 [info] Handle Execution of Cells 2 for /Interactive-1.interactive
14:27:15.071 [info] Kernel acknowledged execution of cell 2 @ 1692912435065
14:27:16.893 [info] End cell 2 execution @ 1692912436891, started @ 1692912435065, elapsed time = 1.826s
14:27:21.255 [info] Generated code for 3 = <ipython-input-3-944b43cc588e> with 4 lines
14:27:21.259 [info] Kernel acknowledged execution of cell 3 @ 1692912441257
14:27:21.850 [info] End cell 3 execution @ 1692912441848, started @ 1692912441257, elapsed time = 0.591s
14:27:21.850 [warn] Cancel all remaining cells due to cancellation or failure in execution
14:30:28.952 [info] Handle Execution of Cells 4 for /Interactive-1.interactive
14:30:28.970 [info] Kernel acknowledged execution of cell 4 @ 1692912628960
14:30:29.009 [info] End cell 4 execution @ 1692912629007, started @ 1692912628960, elapsed time = 0.047s
14:30:29.009 [warn] Cancel all remaining cells due to cancellation or failure in execution
14:30:33.024 [info] Handle Execution of Cells 5 for /Interactive-1.interactive
14:30:33.027 [info] Kernel acknowledged execution of cell 5 @ 1692912633025
14:30:33.168 [info] End cell 5 execution @ 1692912633167, started @ 1692912633025, elapsed time = 0.142s
14:30:36.301 [info] Handle Execution of Cells 6 for /Interactive-1.interactive
14:30:36.307 [info] Kernel acknowledged execution of cell 6 @ 1692912636303
14:30:36.310 [info] End cell 6 execution @ 1692912636308, started @ 1692912636303, elapsed time = 0.005s
14:33:11.847 [info] Generated code for 7 = <ipython-input-7-944b43cc588e> with 4 lines
14:33:11.858 [info] Kernel acknowledged execution of cell 7 @ 1692912791853
14:33:12.351 [info] End cell 7 execution @ 1692912792349, started @ 1692912791853, elapsed time = 0.496s
14:33:12.351 [warn] Cancel all remaining cells due to cancellation or failure in execution
```

**Comment by YUN-MAN** [comment]
I have same problem 🫠
Still don't know how to fix it

**Comment by omicron-8** [comment]
Had the same issue. Downgrading to VSCode 1.81.1 fixed it so the problem might be with VSCode itself

**Comment by YUN-MAN** [comment]
@omicron-8 Thanks a lot! My problem got fix!

**Comment by zatkins2** [comment]
same problem as of 1.82.1 which installed for me a couple days ago

**Comment by zatkins2** [comment]
Seems fixed in 1.82.2?

**Comment by amunger** [comment]
yes, fixed in 1.82.2
https://github.com/microsoft/vscode/issues/192605
